bgpd/bgp_updgrp_packet.c

```diff
@@ -1158,6 +1158,7 @@ void subgroup_default_update_packet(struct update_subgroup *subgrp,
 
 	(void)bpacket_queue_add(SUBGRP_PKTQ(subgrp), s, &vecarr);
 	subgroup_trigger_write(subgrp);
+	subgrp->scount++;
```

Comment:
Shouldn't this be UPDGRP_INCR_STAT?

Reply:
SUBGRP_INCR_STAT?  this shit is confusing why did we do it this way

Reply:
Basically, those MACROS are used for other _stats_, while `scount` is not using MACRO. I think in the feature we could get rid of MACROS which does nothing else than increment/decrement a counter.